packages/core/src/config.js

```diff
@@ -438,6 +438,7 @@ export const comparisonSchema = {
     name: { type: 'string' },
     externalDebugUrl: { type: 'string' },
     domInfoSha: { type: 'string' },
+    metadata: { type: 'object' },
```

Comment:
```suggestion
    metadata: { type: 'object' },
```

define subtype inside `metadata` object is very wide scope.

Reply:
mark that field as optional if it won't be present.

Reply:
Done

Reply:
requires spec update